### PR TITLE
[IMP] oca_dependencies.txt: Add dependency that will need https://git…

### PR DESCRIPTION
**Pull request by martintorresvm**
…hub.com/OCA/business-requirement/pull/4

```
modified:   oca_dependencies.txt
```

<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/OCA/business-requirement/pull/11%23issuecomment-238203549%22%2C%20%22https%3A//github.com/OCA/business-requirement/pull/11%23issuecomment-238213330%22%2C%20%22https%3A//github.com/OCA/business-requirement/pull/11%23discussion_r73862605%22%2C%20%22https%3A//github.com/OCA/business-requirement/pull/11%23issuecomment-238438423%22%2C%20%22https%3A//github.com/OCA/business-requirement/pull/11%23issuecomment-238438889%22%2C%20%22https%3A//github.com/OCA/business-requirement/pull/11%23issuecomment-238685763%22%2C%20%22https%3A//github.com/OCA/business-requirement/pull/11%23issuecomment-238727192%22%2C%20%22https%3A//github.com/OCA/business-requirement/pull/11%23issuecomment-238727499%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/OCA/business-requirement/pull/11%23issuecomment-238203549%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40elicoidal%20%22%2C%20%22created_at%22%3A%20%222016-08-08T10%3A53%3A18Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16461806%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/victormartinelicocorp%22%7D%7D%2C%20%7B%22body%22%3A%20%22%5Cn%5B%21%5BCoverage%20Status%5D%28https%3A//coveralls.io/builds/7349248/badge%29%5D%28https%3A//coveralls.io/builds/7349248%29%5Cn%5CnCoverage%20remained%20the%20same%20at%2090.431%25%20when%20pulling%20%2A%2A28fd694ae5b14be6eed439f7e93de721fc6b7a38%20on%20victormartinelicocorp%3A8.0-add-oca-dependencies-file-project-categ%2A%2A%20into%20%2A%2A25d0d87b7d4816b30054c21c060655f580c9084b%20on%20OCA%3A8.0%2A%2A.%5Cn%22%2C%20%22created_at%22%3A%20%222016-08-08T11%3A49%3A13Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2354108%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/coveralls%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40dreispt%20Thanks%21%22%2C%20%22created_at%22%3A%20%222016-08-09T02%3A46%3A47Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16461806%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/victormartinelicocorp%22%7D%7D%2C%20%7B%22body%22%3A%20%22%5Cn%5B%21%5BCoverage%20Status%5D%28https%3A//coveralls.io/builds/7362584/badge%29%5D%28https%3A//coveralls.io/builds/7362584%29%5Cn%5CnCoverage%20remained%20the%20same%20at%2090.431%25%20when%20pulling%20%2A%2A97f3f91b9a2507bc6c07f31bcc812595627aa392%20on%20victormartinelicocorp%3A8.0-add-oca-dependencies-file-project-categ%2A%2A%20into%20%2A%2A25d0d87b7d4816b30054c21c060655f580c9084b%20on%20OCA%3A8.0%2A%2A.%5Cn%22%2C%20%22created_at%22%3A%20%222016-08-09T02%3A50%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2354108%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/coveralls%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222016-08-09T20%3A46%3A38Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1246629%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dreispt%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20FT%22%2C%20%22created_at%22%3A%20%222016-08-09T23%3A49%3A36Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7600613%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/elicoidal%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222016-08-09T23%3A51%3A36Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16926%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rvalyi%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2028fd694ae5b14be6eed439f7e93de721fc6b7a38%20oca_dependencies.txt%2016%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/OCA/business-requirement/pull/11%23discussion_r73862605%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22You%20just%20need%20name%20of%20the%20dependency%20repo.%5Cr%5CnA%20line%20with%20just%20%60project%60%20is%20enough.%22%2C%20%22created_at%22%3A%20%222016-08-08T11%3A54%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1246629%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dreispt%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20oca_dependencies.txt%3AL1-4%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 28fd694ae5b14be6eed439f7e93de721fc6b7a38 oca_dependencies.txt 16'></a> &nbsp;<b><a href='https://github.com/OCA/business-requirement/pull/11#discussion_r73862605'>File: oca_dependencies.txt:L1-4</a></b>
- <a href='https://github.com/dreispt'></a> You just need name of the dependency repo.
  A line with just `project` is enough.
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/OCA/business-requirement/pull/11#issuecomment-238203549'>General Comment</a></b>
- <a href='https://github.com/victormartinelicocorp'></a> @elicoidal
- <a href='https://github.com/coveralls'></a> [Coverage Status](https://coveralls.io/builds/7349248)
  Coverage remained the same at 90.431% when pulling **28fd694ae5b14be6eed439f7e93de721fc6b7a38 on victormartinelicocorp:8.0-add-oca-dependencies-file-project-categ** into **25d0d87b7d4816b30054c21c060655f580c9084b on OCA:8.0**.
- <a href='https://github.com/victormartinelicocorp'></a> @dreispt Thanks!
- <a href='https://github.com/coveralls'></a> [Coverage Status](https://coveralls.io/builds/7362584)
  Coverage remained the same at 90.431% when pulling **97f3f91b9a2507bc6c07f31bcc812595627aa392 on victormartinelicocorp:8.0-add-oca-dependencies-file-project-categ** into **25d0d87b7d4816b30054c21c060655f580c9084b on OCA:8.0**.
- <a href='https://github.com/dreispt'></a> :+1:
- <a href='https://github.com/elicoidal'></a> :+1: FT

<a href='https://www.codereviewhub.com/OCA/business-requirement/pull/11?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/OCA/business-requirement/pull/11?approve=1'></a>&nbsp;<a href='https://github.com/OCA/business-requirement/pull/11'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dreispt** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/OCA/business-requirement/pull/11'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by martintorresvm** [comment]
@elicoidal

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1,15 +1,3 @@
...
-# To depend on the standard version of sale-workflow, use:
-# sale-workflow
-#
-# To explicitely give the URL of a fork, and still use the version specified in
-# .travis.yml, use:
-# sale-workflow https://github.com/OCA/sale-workflow
-#
-# To provide both the URL and a branch, use:
-# sale-workflow https://github.com/OCA/sale-workflow branchname
+project_categ https://github.com/OCA/project 8.0
```
Comment (dreispt) [inline oca_dependencies.txt]: You just need name of the dependency repo.
A line with just `project` is enough.

**Comment by martintorresvm** [comment]
@dreispt Thanks!

**Comment by dreispt** [comment]
:+1:

**Comment by elicoidal** [comment]
:+1: FT

**Comment by rvalyi** [comment]
:+1:

**Pull request merged**